### PR TITLE
Some boring updates

### PR DESCRIPTION
**Pull request by jfcherng**

### CODE REVIEW

File: HyperClick.sublime-settings [hunk 1]
```diff
@@ -35,7 +35,7 @@
     // - aliases map something short in the import statement (e.g. "@") to a path in your project
     // - HyperClick will also search in lookup_paths, which is particularly useful in project settings
     "scopes": {
-        "source.ts, source.js, source.tsx, source.jsx": {
+        "source.js | source.jsx | source.ts | source.tsx": {
```
Comment (jfcherng) [inline HyperClick.sublime-settings:38]: I think `|` with spaces around has better readability due to extra spaces. But I can revert this if it's not preferred for whatever reason.

I think using dict-order is better than doing it randomly but same, I can revert this if it's not preferred for whatever reason.

File: HyperClick.sublime-settings [hunk 2]
```diff
@@ -81,7 +81,7 @@
         },
         "text.html.basic": {
             "regexes": [
-                ".*?<link\\s+rel=\"import\"\\s+href=['\"](.+)['\"]/?>.*?"
+                ".*?<link\\s+rel=\"import\"\\s+href=['\"](.+)['\"]/?>"
```
Comment (jfcherng) [inline HyperClick.sublime-settings:84]: This trailing `.*?` is totally meaningless and less performant.

File: HyperClick.sublime-settings (hunk 1 above)
Comment (cristianl) [reply HyperClick.sublime-settings:38]: I'm open to this change, but for consistency all other selectors (`scope_is_js` / `scope_is_sass` etc) should be changed

Comment (jfcherng) [reply HyperClick.sublime-settings:38]: resolved in https://github.com/aziz/SublimeHyperClick/pull/78/commits/8e91ba5bd034f691f1fe34f8352149bb7d10a2d0

**Comment by cristianl** [comment]
Released as [2.4.0](https://github.com/aziz/SublimeHyperClick/releases/tag/2.4.0). Thank you @jfcherng!

**Pull request merged**